tensorflow/contrib/losses/python/losses/loss_ops.py

```diff
@@ -664,3 +665,60 @@ def cosine_distance(
     radial_diffs = math_ops.multiply(predictions, labels)
     losses = 1 - math_ops.reduce_sum(radial_diffs, reduction_indices=[axis,])
     return compute_weighted_loss(losses, weights, scope=scope)
+
+
+def focal_loss(predictions, labels, gamma=2, alpha=1, weights=1.0,
```

Comment:
The arguments to losses should be labels,predictions instead of predictions,labels